### DISCUSSION TITLE
Dashboard manual removal

### DISCUSSION THREAD

**Discussion by a-atalla** [mirumee/saleor-dashboard#3399] [General] [answered]
I remember i saw this manual before, but today i needed to go through it and i find out that it was completely removed from the docs https://github.com/saleor/saleor-docs/pull/610/commits/815180415e191e68c7c9526ef5094594e2e1555b.
Wondering what is the reason behind such a decision?

**Comment by karolkielecki** [comment, marked as answer]
Hi @a-atalla, we've removed Dashboard Manual. It needed to be updated in some places, but we'll provide a better alternative. Could you please tell us what you're looking for?

**Comment by TurkaljGoran** [comment]
Hello, when can we expect the new Dashboard Manual?
